Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -2018,7 +2018,7 @@ static URL conformFragmentIdentifierForURL(const URL& url)
 #if ENABLE(LEGACY_ENCRYPTED_MEDIA)
 static bool keySystemIsSupported(const String& keySystem)
 {
-    if (equalIgnoringASCIICase(keySystem, "com.apple.fps") || equalIgnoringASCIICase(keySystem, "com.apple.fps.1_0") || equalIgnoringASCIICase(keySystem, "org.w3c.clearkey"))
+    if (equalLettersIgnoringASCIICase(keySystem, "com.apple.fps") || equalIgnoringASCIICase(keySystem, "com.apple.fps.1_0") || equalLettersIgnoringASCIICase(keySystem, "org.w3c.clearkey"))
```

Comment:
I think we should use just return here instead if return true; return false

Reply:
Fixed, thanks.